api/proto/teleport/legacy/types/types.proto

```diff
@@ -6536,3 +6536,21 @@ message OktaOptions {
     (gogoproto.casttype) = "Duration"
   ];
 }
+
+// AccessGraphSync is a configuration for Access Graph service.
+message AccessGraphSync {
+  repeated AccessGraphAWSSync AWS = 1 [(gogoproto.jsontag) = "types,omitempty"];
+}
+
+// AccessGraphAWSSync is a configuration for AWS Access Graph service poll service.
+message AccessGraphAWSSync {
+  // AccountID is the AWS account ID to sync.
+  string AccountID = 1 [(gogoproto.jsontag) = "account_id,omitempty"];
```

Comment:
Do we need to ask for the account id?
It usually comes from the credentials available.